src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CohortController.cs

```diff
@@ -168,9 +173,32 @@ public IActionResult AddDraftApprenticeship2(AddDraftApprenticeshipViewModel mod
         [Authorize(Policy = nameof(PolicyNames.HasContributorOrAbovePermission))]
         public async Task<IActionResult> SelectDeliveryModel(AddDraftApprenticeshipViewModel request)
         {
-            return View("SelectDeliveryModel", request);
+            var models = (await GetProviderCourseDeliveryModels(request.ProviderId, request.CourseCode)).ToArray();
```

Comment:
Is there any benefit in converting it to an array in this method?  We don't appear to be using any of the array features.

Reply:
to avoid multiple enumerations warnings

Reply:
This is probably one of those times where the warning is wrong, but it's not going to make any realistic difference either way.